src/fund/types/events.ts

```diff
@@ -0,0 +1,40 @@
+export type OpenEvent = {
+  eventName: 'open';
+  widgetName: string;
+};
+
+export type TransitionViewEvent = {
+  eventName: 'transition_view';
+  pageRoute: string;
+};
+
+export type PublicErrorEvent = {
+  eventName: 'error';
+  // TODO: Public error shape
```

Comment:
@rustam-cb we shouldn't just blindly copy-paste TODO comments from our old SDK. We can probably just delete this.